labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -58,9 +49,7 @@ object ListManipulationExercise01 {
    *  - custom made
    *  - ... etc 
    */
-  def concatLists[T](l1: List[T], l2: List[T]): List[T] = {
-    error("fix me")
-  }
+  def concatLists[T](l1: List[T], l2: List[T]): List[T] = { l1:::l2 }
```

Comment:
`++` is a syntax that works for any collection, as opposed to `:::` which is somewhat of a legacy syntax. For lists they do they same thing though.